### PR TITLE
Add initial files

### PR DESCRIPTION
**Pull request by misraved**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+## v0.10.0 [2023-03-22]
```
Comment (cbruno10) [inline CHANGELOG.md:1]: We can probably clear out all previous CHANGELOG options if we're using the next minor version from the Turbot plugin

File: README.md [hunk 2]
```diff
@@ -1,2 +1,89 @@
-# steampipe-plugin-guardrails
-Use SQL to instantly query the Turbot Guardrails resources. Open source CLI. No DB required.
+![image](https://hub.steampipe.io/images/plugins/turbot/guardrails-social-graphic.png)
+
+# Turbot Guardrails Plugin for Steampipe
+
+Use SQL to query infrastructure including servers, networks, identity and more from Turbot Guardrails.
```
Comment (cbruno10) [inline README.md:5]: Is this description correct? I think we'd want to mention different resources

File: config/guardrails.spc [hunk 3]
```diff
@@ -0,0 +1,17 @@
+connection "turbot" {
+  plugin = "turbot"
+
+  # Default credentials:
+  # By default, Steampipe will use your Turbot profiles and credentials exactly
```
Comment (cbruno10) [inline config/guardrails.spc:5]:
```suggestion
  # By default, Steampipe will use your Turbot Guardrails profiles and credentials exactly
```

I think we'd probably need similar updates here and in all other files/comments as well

File: config/guardrails.spc [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  # the same as the Turbot CLI and Turbot Terraform provider. In many cases, no
+  # extra configuration is required to use Steampipe.
+
+  # Use an existing Turbot profile configured in ~/.config/turbot
```
Comment (cbruno10) [inline config/guardrails.spc:9]: Is this changing in the CLI? Assuming not, but worth checking

File: docs/index.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+display_name: Turbot Guardrails
+short_name: guardrails
+description: Steampipe plugin to query resources, controls, policies and more from Turbot Guardrails.
+og_description: Query Turbot Guardrails with SQL! Open source CLI. No DB required.
+og_image: "/images/plugins/turbot/guardrails-social-graphic.png"
+---
+
+# Turbot Guardrails + Steampipe
+
+[Turbot](https://turbot.com/) is a cloud governance and security platform with a real-time CMDB for cloud resources.
```
Comment (cbruno10) [inline docs/index.md:15]:
```suggestion
[Turbot Guardrails](https://turbot.com/guardrails) is the leading platform for policy-based control and automatic remediation of enterprise clouds.
```

Description taken from the site

File: docs/index.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+}
+
+```
+
+### Credentials from environment variables
+
+Environment variables provide another way to specify default Turbot Guardrails CLI credentials:
+
+```sh
+export TURBOT_GUARDRAILS_SECRET_KEY=3d397816-575f-4b2a-a470-a96abe29b81a
```
Comment (cbruno10) [inline docs/index.md:112]: Are these the new env vars the CLI is going to use too?

File: docs/index.md [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+export TURBOT_GUARDRAILS_ACCESS_KEY=86835f29-1c88-46d9-b6ce-cbe5016842d3
+export TURBOT_GUARDRAILS_WORKSPACE=https://turbot-acme.cloud.turbot.com
+```
+
+You can also change the default profile to a named profile with the TURBOT_PROFILE environment variable:
+
+```sh
+export TURBOT_PROFILE=turbot-acme
```
Comment (cbruno10) [inline docs/index.md:120]: Is this going to be TURBOT_GUARDRAILS_PROFILE or remain the same (or be something else) in the CLI?

File: docs/tables/guardrails_control.md [hunk 8]
```diff
@@ -0,0 +1,162 @@
+# Table: guardrails_control
+
+Controls in Turbot represent the state of a given check (control type) against
```
Comment (cbruno10) [inline docs/tables/guardrails_control.md:3]:
```suggestion
Controls in Turbot Guardrails represent the state of a given check (control type) against
```

File: guardrails/table_guardrails_active_grant.go [hunk 9]
```diff
@@ -0,0 +1,186 @@
...
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+func tableTurbotActiveGrant(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "guardrails_active_grant",
+		Description: "All active grants of resources by Turbot.",
```
Comment (cbruno10) [inline guardrails/table_guardrails_active_grant.go:17]:
```suggestion
		Description: "All active grants of resources by Turbot Guardrails.",
```

File: guardrails/table_guardrails_active_grant.go [hunk 10]
```diff
@@ -0,0 +1,186 @@
...
+	"fmt"
+	"regexp"
+	"strconv"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+func tableTurbotActiveGrant(ctx context.Context) *plugin.Table {
```
Comment (cbruno10) [inline guardrails/table_guardrails_active_grant.go:14]:
```suggestion
func tableGuardrailsActiveGrant(ctx context.Context) *plugin.Table {
```

File: guardrails/types.go [hunk 11]
```diff
@@ -0,0 +1,626 @@
...
+type ResourcesResponse struct {
+	Resources struct {
+		Items  []Resource
+		Paging struct {
+			Next string
+		}
+	}
+}
+
+type TurbotIDObject struct {
```
Comment (cbruno10) [inline guardrails/types.go:14]:
```suggestion
type GuardrailsIDObject struct {
```

**Review by cbruno10 — CHANGES_REQUESTED** [review]
@misraved Please see review comments. I didn't comment on all instances of changing Turbot -> Guardrails or adding the word "Guardrails" in front of "Turbot", but added some comments that can be extrapolated to other instances.

File: config/guardrails.spc (hunk 4 above)
Comment (misraved) [reply config/guardrails.spc:9]: Yes, the CLI  has also been rebranded to Turbot Guardrails CLI - https://turbot.com/guardrails/docs/releases/cli

File: docs/index.md (hunk 7 above)
Comment (misraved) [reply docs/index.md:120]: The env variables haven't changed. I will revert [Update env variable references](https://github.com/turbot/steampipe-plugin-guardrails/pull/1/commits/5a44fe340709686762184d7e7984790c9cdde525) commit.

**Review by cbruno10 — APPROVED** [review]

File: CHANGELOG.md [hunk 12]
```diff
@@ -0,0 +1,19 @@
+## v0.11.0 [2023-07-27]
+
+_What's new?_
+
+- The [Turbot](https://hub.steampipe.io/plugins/turbot/turbot/tables) tables have now been rebranded to use `Guardrails` instead:
```
Comment (vkumbha) [inline CHANGELOG.md:5]: should this be https://hub.steampipe.io/plugins/turbot/guardrails/tables ?

**Pull request merged**